From patchwork Thu Apr 22 16:38:33 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Brijesh Singh <brijesh.singh@amd.com>
X-Patchwork-Id: 12218913
Return-Path: <kvm-owner@kernel.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-18.8 required=3.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,HEADER_FROM_DIFFERENT_DOMAINS,INCLUDES_CR_TRAILER,
	INCLUDES_PATCH,MAILING_LIST_MULTI,MSGID_FROM_MTA_HEADER,SPF_HELO_NONE,
	SPF_PASS,USER_AGENT_GIT autolearn=ham autolearn_force=no version=3.4.0
Received: from mail.kernel.org (mail.kernel.org [198.145.29.99])
	by smtp.lore.kernel.org (Postfix) with ESMTP id DB451C433ED
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 16:39:50 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id B9C6761409
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 16:39:50 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S238011AbhDVQkY (ORCPT <rfc822;kvm@archiver.kernel.org>);
        Thu, 22 Apr 2021 12:40:24 -0400
Received: from mail-co1nam11on2057.outbound.protection.outlook.com
 ([40.107.220.57]:61792
        "EHLO NAM11-CO1-obe.outbound.protection.outlook.com"
        rhost-flags-OK-OK-OK-FAIL) by vger.kernel.org with ESMTP
        id S236627AbhDVQkW (ORCPT <rfc822;kvm@vger.kernel.org>);
        Thu, 22 Apr 2021 12:40:22 -0400
ARC-Seal: i=1; a=rsa-sha256; s=arcselector9901; d=microsoft.com; cv=none;
 b=RK2UQYFA2DrXKkQ+m91Dra79rZ23uTGHb68bz3IXwQnHCv52GzCG/eLr37utKO5H6R5SZUgUXguMHGKbwy4l7Y1o0Zcu6naSXR821gH+HAVQYET2WuGpzMnO3VbuMOtd1zOrCTP2hYFx1hKNVhffGSGjWscZLKqNE1eDXp0soiCJm988pZzGgne67wBbmTWAJMWx5ScZzIIaIMvqO8GMDVmdI6m4TYn9KruKGe1uaPSmoRTXl0qolRorLXOiyX76RCYN8jpCszKchtPyRn0wEcHqPoUK2n3dbfdONSo9rkdY5oS1l6luI6vkmPdi7uXoeIxAH0yjoOGg9c9XYilIiA==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=microsoft.com;
 s=arcselector9901;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-SenderADCheck;
 bh=suPWH9bl0K1+mahnGBn6QWxHfYVbSDId4Kaw+uxxWog=;
 b=nBvwEXHYF5BYOSHCI+XuG2CNK0p8PkBpYRwYQTxTN/4lGn+rcmWqYX6hauswKNvZcd8i0HKd+vTyEO8UShbvJCdIoGaXp8FEHlMLwYMxU4Cu+4vyqVuW/xhvR2xvqbI8w++WZyqrpUDh5NzpcKi8QTxfQFR5KOGdhgDxW2L2Fq2H9lA2RL9QSlBtuJn/PW4wKdncgXWis3PkzyDvYVMWoP4+ZTVpaVc68MGBFnlhXElB63PRPt4MA/KBNj8BFh5QKnJvhaz+t0LVdyfmuAdwsq7hiDRzEFf35UKd8m+du0JMVHyEIBftgFIDUlUYliN1uyEFTNm5fUzBpkPNwp2ixQ==
ARC-Authentication-Results: i=1; mx.microsoft.com 1; spf=pass
 smtp.mailfrom=amd.com; dmarc=pass action=none header.from=amd.com; dkim=pass
 header.d=amd.com; arc=none
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=amd.com; s=selector1;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-SenderADCheck;
 bh=suPWH9bl0K1+mahnGBn6QWxHfYVbSDId4Kaw+uxxWog=;
 b=b1RwjPw1a5hFd9j/qV9NNZMwzzaQ2vpArt5pg1EkbJtdYGcOr5ZLENc2+/UNCv/AtnqmUMf29qMvCtd00/bFtMmbJBuZjyOwwofHx88uAY9UL9K9rTryq3AfE+ZhMEgj/B+tDcPqsR5BrUqBc+ArGC7EFLx2QgfkE/fVMX/l8CQ=
Authentication-Results: redhat.com; dkim=none (message not signed)
 header.d=none;redhat.com; dmarc=none action=none header.from=amd.com;
Received: from SN6PR12MB2718.namprd12.prod.outlook.com (2603:10b6:805:6f::22)
 by SA0PR12MB4495.namprd12.prod.outlook.com (2603:10b6:806:70::13) with
 Microsoft SMTP Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.4042.19; Thu, 22 Apr
 2021 16:39:46 +0000
Received: from SN6PR12MB2718.namprd12.prod.outlook.com
 ([fe80::9898:5b48:a062:db94]) by SN6PR12MB2718.namprd12.prod.outlook.com
 ([fe80::9898:5b48:a062:db94%6]) with mapi id 15.20.4065.023; Thu, 22 Apr 2021
 16:39:46 +0000
From: Brijesh Singh <brijesh.singh@amd.com>
To: pbonzini@redhat.com, kvm@vger.kernel.org
Cc: linux-kernel@vger.kernel.org, sfr@canb.auug.org.au,
        Ashish.Kalra@amd.com, Brijesh Singh <brijesh.singh@amd.com>
Subject: [PATCH 1/4] docs: kvm: fix underline too short warning for
 KVM_SEV_RECEIVE_START
Date: Thu, 22 Apr 2021 11:38:33 -0500
Message-Id: <20210422163836.27117-2-brijesh.singh@amd.com>
X-Mailer: git-send-email 2.17.1
In-Reply-To: <20210422163836.27117-1-brijesh.singh@amd.com>
References: <20210422163836.27117-1-brijesh.singh@amd.com>
X-Originating-IP: [165.204.77.1]
X-ClientProxiedBy: SN2PR01CA0059.prod.exchangelabs.com (2603:10b6:800::27) To
 SN6PR12MB2718.namprd12.prod.outlook.com (2603:10b6:805:6f::22)
MIME-Version: 1.0
X-MS-Exchange-MessageSentRepresentingType: 1
Received: from sbrijesh-desktop.amd.com (165.204.77.1) by
 SN2PR01CA0059.prod.exchangelabs.com (2603:10b6:800::27) with Microsoft SMTP
 Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.4065.20 via Frontend
 Transport; Thu, 22 Apr 2021 16:39:45 +0000
X-MS-PublicTrafficType: Email
X-MS-Office365-Filtering-Correlation-Id: 2cb90bb3-e98a-40e6-f541-08d905ad3cc5
X-MS-TrafficTypeDiagnostic: SA0PR12MB4495:
X-MS-Exchange-Transport-Forked: True
X-Microsoft-Antispam-PRVS: 
 <SA0PR12MB4495139D6D3A33B9349D6540E5469@SA0PR12MB4495.namprd12.prod.outlook.com>
X-MS-Oob-TLC-OOBClassifiers: OLM:4714;
X-MS-Exchange-SenderADCheck: 1
X-Microsoft-Antispam: BCL:0;
X-Microsoft-Antispam-Message-Info: 
 Zvsn9QEoe4xcP3av5gp6yHnC3T18+wS7FjZWd0wPYkEcaXzm+l0BOmjNtugoeY/FT09Rk0lgXFi4Zh5XhOMS+le0Tjz+xuyGHINQfZzoOSDi9aHtQRtLOyG4z1uwKBe1YgkKfkKSwBTyw0XwLqq5swxX3Fd+uAuDByxUHA8aBZj/aaQM43olaKlEOrpefXnYymaw5eHNidFCxbiNX7UK+NEq/dIASVOE5l8iWBc7qK/E/JSWIImwlOu1/nh3qcLpb/rxaDcqGU28nT/q8UAupbv0FLCchj95TcPXMdBF7LXkBrKWYLiR/KxZ+VClNczUGQ5vQVhWchYJa3InmhfFvOhrOVS+VMPV44A7sW5zuGpy7m33beoIMXf0EHp6qdHdoeQrecQrGCPocpXaXwiONKw6VGrWbg/AasZZujESOfDT63wmhjx7GeVVs5Q5vkZnIr2d/XOrAZgWWXt/qzi6czLwyq+yTjPgKo9Q5Y5CiDuRsKiTSrEu13TZfCoMd/jH/qjbFSvCvIusu0J5tL6MeNJFCpm8XRVz09yUQkuYTA8Xg8js7O3bPE5XDQowxsjrif3znsDeaVNBbvDRHwsNgL6ENHnXfZa8pusVjFRK/bbXYjPBsLF/SA9LcGYBJk/AO1vl61JK5D6azIHYam0uZsLShyi+T4Afw013oWuQUQc=
X-Forefront-Antispam-Report: 
 CIP:255.255.255.255;CTRY:;LANG:en;SCL:1;SRV:;IPV:NLI;SFV:NSPM;H:SN6PR12MB2718.namprd12.prod.outlook.com;PTR:;CAT:NONE;SFS:(4636009)(376002)(136003)(366004)(346002)(396003)(39850400004)(26005)(8676002)(66476007)(6486002)(4744005)(956004)(1076003)(16526019)(2616005)(36756003)(478600001)(4326008)(5660300002)(186003)(83380400001)(316002)(44832011)(38100700002)(38350700002)(6666004)(7696005)(66556008)(2906002)(86362001)(66946007)(8936002)(52116002);DIR:OUT;SFP:1101;
X-MS-Exchange-AntiSpam-MessageData: 
 tBDC5xr8cB4SRcKTLw4DCPkaz7tonT/1Ta4hbtMxpjsp4BEoOdQzJ9S4hGXhZ0fMbW77FkotiSdtJbfcAfAAwL9QIDZeA743eXz1VxDN5QrhRIiBHybtLyJesiihyd20AD/Ogg0ZgOqG4pHgqmhIk7sbgvivFiMotxqAc3+t7Lu9n4xddSyPM8HRxjwSy1kz3QeNMWqR5Te6ewDRT466e0kmg9XtLlA8qjuRruKef8c4sS4+tQjBaicPdyGsIIGYGprHYVD6rhAET9G7VKVEGZyA+cqnGPCoX5564m/l3ShbEXaKNMQzCljA6S7vAY+VohS7ZQW+mVym5NAQzKzjZ9fWRVf/8/KhqId0fM9f3UmzP/SpqF2+DLmQXn2DrIbnJzVeFOCxF5XPvPtfur+JeeycNHhEmoYuP5pGidxRwC2qxOjtszAuWco/ATmCx7oF3jY/n7AtukB8WCXW/+0COWshgo24SeP49nKkSpq86k0LFTVI30G8My4Fxge9+Fe01MZQZwhvGZZOYgnJf9jAsZi1dCR/qsIn+VgEmdVc2h73lAWKX/imz+z8p9icX3fppZEi8jXvuozZvSLeySY8ftiX09ZUfEGDbmQYk50BqWEksZCQg1o3BNETY53z+3Xnp0wXVBqqvlfyvHQ/Jqw6qaVf0ysJnPY0qZH7rVwU08JvcrYykI4rJtyM3+XySXRhLf+9EfRHq3rUA/JXj7UEDPMR+NAWKfpn9w7qsK05yW5GuSGfV4Sw55g82jtfbxLF+Sa7UtwMkW9HP/2CSPoJ1ohqKbLraWx4SAu+HFo3mOyFnGhM53Gt0lt9xXiJUEgOcRIvIpDM0xBO3/1WWcWF7p1zoM9gC1tMSwsXRYTcj7c7E0d8T9jBuImD8IieRJ7X766c7/IoHDkEBeiiLb3/dErMPDRfR0QBrFwaC2jjoOEo5a8Sy9/AWbphGOdj3iUpwqJpewsKsFE4rHC1/G4WbRlDn2AP4qH2bel9IFINfJD4k/jM4dax1xwZ8qXgyN4RQ2uZbC7yS8azsgFwdSUmqvUBkDq6m42ZWYelBZWO2+0PWkTqLRJfNJLjtIrdMF5meusjk6cdxA6dAr+fOJVJZ8wbzGyNF+VsedE/SOsQTiwphLcw8cydwMjUoPXWjEqm3Ym8BN9BGZWcSzoSIsPvdY8uH0h8cyJbALBSl+BKblBPxFBVP5N7XiocmeFoStIF6QQJPdzjMJwowFQFxjkz0OcwqUz8GPPICnZk/c77qg/gQu5Ix4v8M+e2ps4T+vS022oNEnNhWDm2+F5FeO2s/RDvArNkG4yt1hL4qlEEU/Fm+y60Q0G9vbGPRe/3WOjC
X-OriginatorOrg: amd.com
X-MS-Exchange-CrossTenant-Network-Message-Id: 
 2cb90bb3-e98a-40e6-f541-08d905ad3cc5
X-MS-Exchange-CrossTenant-AuthSource: SN6PR12MB2718.namprd12.prod.outlook.com
X-MS-Exchange-CrossTenant-AuthAs: Internal
X-MS-Exchange-CrossTenant-OriginalArrivalTime: 22 Apr 2021 16:39:45.7962
 (UTC)
X-MS-Exchange-CrossTenant-FromEntityHeader: Hosted
X-MS-Exchange-CrossTenant-Id: 3dd8961f-e488-4e60-8e11-a82d994e183d
X-MS-Exchange-CrossTenant-MailboxType: HOSTED
X-MS-Exchange-CrossTenant-UserPrincipalName: 
 ivVo258cHORq6Kj1lsO4ESenGFZKkrCUERpbwOeteb1zwbv0le7zdl6Cha5u0sNl7xVh6hGUxhGimoXYaiauOQ==
X-MS-Exchange-Transport-CrossTenantHeadersStamped: SA0PR12MB4495
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Fixes: af43cbbf954b ("KVM: SVM: Add support for KVM_SEV_RECEIVE_START command")
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Brijesh Singh <brijesh.singh@amd.com>
---
 Documentation/virt/kvm/amd-memory-encryption.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/virt/kvm/amd-memory-encryption.rst b/Documentation/virt/kvm/amd-memory-encryption.rst
index 907adfe60090..07f8633b61cd 100644
--- a/Documentation/virt/kvm/amd-memory-encryption.rst
+++ b/Documentation/virt/kvm/amd-memory-encryption.rst
@@ -366,7 +366,7 @@ migration can restart with a new target later.
 Returns: 0 on success, -negative on error
 
 15. KVM_SEV_RECEIVE_START
-------------------------
+--------------------------
 
 The KVM_SEV_RECEIVE_START command is used for creating the memory encryption
 context for an incoming SEV guest. To create the encryption context, the user must

From patchwork Thu Apr 22 16:38:34 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Brijesh Singh <brijesh.singh@amd.com>
X-Patchwork-Id: 12218917
Return-Path: <kvm-owner@kernel.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-18.8 required=3.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,HEADER_FROM_DIFFERENT_DOMAINS,INCLUDES_CR_TRAILER,
	INCLUDES_PATCH,MAILING_LIST_MULTI,MSGID_FROM_MTA_HEADER,SPF_HELO_NONE,
	SPF_PASS,USER_AGENT_GIT autolearn=ham autolearn_force=no version=3.4.0
Received: from mail.kernel.org (mail.kernel.org [198.145.29.99])
	by smtp.lore.kernel.org (Postfix) with ESMTP id 37EC3C433ED
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 16:39:53 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 120486141C
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 16:39:53 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S238106AbhDVQk0 (ORCPT <rfc822;kvm@archiver.kernel.org>);
        Thu, 22 Apr 2021 12:40:26 -0400
Received: from mail-co1nam11on2057.outbound.protection.outlook.com
 ([40.107.220.57]:61792
        "EHLO NAM11-CO1-obe.outbound.protection.outlook.com"
        rhost-flags-OK-OK-OK-FAIL) by vger.kernel.org with ESMTP
        id S237920AbhDVQkX (ORCPT <rfc822;kvm@vger.kernel.org>);
        Thu, 22 Apr 2021 12:40:23 -0400
ARC-Seal: i=1; a=rsa-sha256; s=arcselector9901; d=microsoft.com; cv=none;
 b=RFSP5HpYYYLmcwLqZqPB6Tqmm+iykQMQ5dbxMUE6S1BsW0YAXrI5hWsaSlwYVtSIupdMvq21htDdcrukRuHCWQeiVjomi8Y7r9Y0tKtAYTNy4JZa7FuE6AwhI5zPE8DoKkKcd3DrAbdd/Qu7pdlMSLZxhcJ4mecbvOYVlo9wRT0/MULINDvxVazowIL0csCdEZi+ZHZB8OBKggKp/m++dg8G4zRuqq8+RH5nzA1PWey+FZS/tqF72uwkCNqhEr2zG/71dirBwJ03bxzZzOYoJ51NTZOjgOBs30XM/RJU2B8mtGpDxiVXCHYMSsa3aHkY0NeOv/pio6IXm7ozQUMfOA==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=microsoft.com;
 s=arcselector9901;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-SenderADCheck;
 bh=rKW8Betutlx583j2ne8bboN+lfkQNHJNO7OIEIj6B/c=;
 b=bxE1sAUCChnece2rcIAbojRMU1kjndHpa3/QlrJow1e6YZhQCTwQRDqd57pc2WPkhaMGgIYI1bVlja0Y2ihrsLgVpYtgTrZE+Vzy8t2Oca/okyqnxtubWT36BhCDiAfqOU8QhHOgJ4qVgWKnL/3HlWn1RFmoWqaXjOs93szhKZW6NR6f/V2FKmzhvWSH9rpSadnkOB0SQ+Znub4srgNbhhpCPYGl/bh9Ww5+02VZ1m8q9jW8X3nvSln8YyJ/yUOU1pMmaF/EuegZCdyZ12qrHyqDZ1bBjV0AXzAj+qEz4KMDa+rf57tk6xs/GwQZ5wBA6Bn5ntUrDsNq1auGHwstAA==
ARC-Authentication-Results: i=1; mx.microsoft.com 1; spf=pass
 smtp.mailfrom=amd.com; dmarc=pass action=none header.from=amd.com; dkim=pass
 header.d=amd.com; arc=none
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=amd.com; s=selector1;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-SenderADCheck;
 bh=rKW8Betutlx583j2ne8bboN+lfkQNHJNO7OIEIj6B/c=;
 b=YTlCm5poUPpnvfcw7piZ8c60O5HaZam7MrAXRYQlvueMVLVqffTqPP3GEzXhpLWMn7L6I9N1seoq0EKZBPSVAk2FjDIArC/CoAyKUxXgcW/LnrhXHD9jpHhgI+d25OJPLsNptEH/s6u4a3Xl2IejvAIt4NVhbEvXFV6PeEfL0zY=
Authentication-Results: redhat.com; dkim=none (message not signed)
 header.d=none;redhat.com; dmarc=none action=none header.from=amd.com;
Received: from SN6PR12MB2718.namprd12.prod.outlook.com (2603:10b6:805:6f::22)
 by SA0PR12MB4495.namprd12.prod.outlook.com (2603:10b6:806:70::13) with
 Microsoft SMTP Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.4042.19; Thu, 22 Apr
 2021 16:39:46 +0000
Received: from SN6PR12MB2718.namprd12.prod.outlook.com
 ([fe80::9898:5b48:a062:db94]) by SN6PR12MB2718.namprd12.prod.outlook.com
 ([fe80::9898:5b48:a062:db94%6]) with mapi id 15.20.4065.023; Thu, 22 Apr 2021
 16:39:46 +0000
From: Brijesh Singh <brijesh.singh@amd.com>
To: pbonzini@redhat.com, kvm@vger.kernel.org
Cc: linux-kernel@vger.kernel.org, sfr@canb.auug.org.au,
        Ashish.Kalra@amd.com, Brijesh Singh <brijesh.singh@amd.com>
Subject: [PATCH 2/4] docs: kvm: fix underline too sort warning for
 KVM_SEV_RECEIVE_UPDATE_DATA
Date: Thu, 22 Apr 2021 11:38:34 -0500
Message-Id: <20210422163836.27117-3-brijesh.singh@amd.com>
X-Mailer: git-send-email 2.17.1
In-Reply-To: <20210422163836.27117-1-brijesh.singh@amd.com>
References: <20210422163836.27117-1-brijesh.singh@amd.com>
X-Originating-IP: [165.204.77.1]
X-ClientProxiedBy: SN2PR01CA0059.prod.exchangelabs.com (2603:10b6:800::27) To
 SN6PR12MB2718.namprd12.prod.outlook.com (2603:10b6:805:6f::22)
MIME-Version: 1.0
X-MS-Exchange-MessageSentRepresentingType: 1
Received: from sbrijesh-desktop.amd.com (165.204.77.1) by
 SN2PR01CA0059.prod.exchangelabs.com (2603:10b6:800::27) with Microsoft SMTP
 Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.4065.20 via Frontend
 Transport; Thu, 22 Apr 2021 16:39:45 +0000
X-MS-PublicTrafficType: Email
X-MS-Office365-Filtering-Correlation-Id: cd4f7468-6e0e-4681-202e-08d905ad3d0a
X-MS-TrafficTypeDiagnostic: SA0PR12MB4495:
X-MS-Exchange-Transport-Forked: True
X-Microsoft-Antispam-PRVS: 
 <SA0PR12MB4495922ECB84508B09C72A8BE5469@SA0PR12MB4495.namprd12.prod.outlook.com>
X-MS-Oob-TLC-OOBClassifiers: OLM:2657;
X-MS-Exchange-SenderADCheck: 1
X-Microsoft-Antispam: BCL:0;
X-Microsoft-Antispam-Message-Info: 
 aaUjN+75853zapLG8l/UtlVD6ru+TWTdBzikMA451rXS43dbzsq/2+YSNSLYyJjbliLU5jC64sLbbpv1krtXk9p2dPSwwrKvkO9YobRiQM2azfgbV/iuT/FpQEmJVnoRh6hZbPEOtUTQcm6fytuHH18ycVKAX0suIwIjdwezVNTIBOgT5jHWlR8V6KBdolg4R+b23Rf+/TMsmcajQSMHpwT1zfX/4aiIhdwBH1c8eHcITz5oqsO5bG/6TFRi/PwW4XZmNGPG5Ae5Y8JD/q5SliS+vJ0ijrJx0FXIpnyslqv7vh8ytQ9+e8ce+PxqLkWi6Ajp/FqN3V/RrzxogQdy0SnVSal6UOSf7sTOX+7f4FlfA3WGsusnBhmZcKpfkITQ4CzirVQoyuj0WBbftenQQUudlFAe+E4gWM1eXTNy3Zfw3mB6/LPFXp+DbnAkv5SdcUVt0KmkbVq1NO6BPjtsqrTw9j9vc71dcXTfxrbgCQDXtEx39WU/6uCNVRt0MMCf1/eYm6pDJlXG6mGtacGR5aR99tuMrDYpQRQi1gerxopb8fZtrOFrOvwk0Ac3JVkv7IUy0eXrREctZ+H7kh3gvd6UvfL+IqBkBAn0H9sulM27oQARf9l4i27X5bYMLO6oKwTlJ4Vad7N5mF653n7HkSmPock5wUjqkFLgGWyul90=
X-Forefront-Antispam-Report: 
 CIP:255.255.255.255;CTRY:;LANG:en;SCL:1;SRV:;IPV:NLI;SFV:NSPM;H:SN6PR12MB2718.namprd12.prod.outlook.com;PTR:;CAT:NONE;SFS:(4636009)(376002)(136003)(366004)(346002)(396003)(39850400004)(26005)(8676002)(66476007)(6486002)(4744005)(956004)(1076003)(16526019)(2616005)(36756003)(478600001)(4326008)(5660300002)(186003)(83380400001)(316002)(44832011)(38100700002)(38350700002)(6666004)(7696005)(66556008)(2906002)(86362001)(66946007)(8936002)(52116002);DIR:OUT;SFP:1101;
X-MS-Exchange-AntiSpam-MessageData: 
 QakPFCdbprdZgYufxlgUmgqHITAxLvQFrbdtWmyqLl/9gW8aOtL6WjvomIR4uYeQLs0PFm4JF8qzRSs3MwKF0R80S8dZq1E44OAfH7EHqILjpi/5VjpOe794M7EDdER+7N+tFxjgbUBhgnHYHmU+lhVZRB2BioxqAmksjua7yMnJVUifujOv2MfpYUSPxSGj/8Pt5wHxjxTN5cHp5SiE17CVHJ3g9FXNtp6ygEIhmD/a4LWKvVqDrh23LAhncSQmO90cT8nn5AViqT/3qgHbEE0GJED2arKlET+w24lv+DdnB3ADNOn//VRD+ymToBxKGVw1pD7MuM8unMGrSUgzTksjg8mT1B17oOkbH1Mlt0PYIRpF4kmuqFs1l8NQof+GjPiiYcmryVM2pb7pf46DGCsEZMDguxQ0kM/2ztfagE9mwLzu51zSw3T47Bz3eJbw9nVT49xFYJQHe6UYuObTcxbuv2iV0r+s8uORP8eEwH9VXSqFe7yMEVH3Doeym38bTbb38BYRjz+QBT71v5do4zCkuQUrjar0BczHA9aonjh9G0Cz4rVyNdPUbkwJVzdTxdYXnQJajovKDDm5rFOMsbiMPCc/1RVREmygshlgMCjj9aKFzXS58LSfwRpHqY9IBNnaU+s5wPfO45E5fc6T/6OdP+RKLsUmM9cCLYz4WeM5u0gwcgoDh5VKtYCE82rUlSL0i6pFVJVe8MbhovBFtG9KrjMDYKfDXuIXz1M1wwk76bzqWPQKf5EYz2GHH0v/G6TISwtGq/Csgg4c8GVI3jlZysfwyN3/iPZkkk6XdwkeTDMk182E6a5d4sXKU0hNgNS8wyyd1VDxvDXm5Ch+YJ7r+w376qoOStgA1MAvN87IaMk4USFUFUogbUFGJp2103QirGaSeAnh/VCpnARhvfVNy8ueOl0iMQq0M7zCWgBiKJrIBD/d6YlONgLqD+7i0489OJcv0AtpOsftrBZuQFIucljHovNXjnbeZBk3w3eMmLx4SfqkP0FXP9FU3CVs6GZUXhI5RINWmkdzCEBzJKDXSBhfG95EzvsWZmgFPENy8yKQUH6huDoDucMir1V5pMfHPWvxcH6ntY7Yo9cXx4o34VMS+kHci6JyOULkxJehCqM5AL7vnDXg9IaO/NDPfjem9Y2V2S7gudxXkhSzX9AxwkptNvgIlhJxA4ovSfxjhc4qrSkgud53BQWE51TxS1Jad+NjbjKKOfZln3EikTitczXLwdJOZ3bNLRuVcgr4byDRxcQpz2OlfHsVbEP7i9rKEQ/jxR+PVTSxgUrkq0G8xGQbGxzuOHDzYp8lz/wKkGP6yo1v3tLyxigmJKoD
X-OriginatorOrg: amd.com
X-MS-Exchange-CrossTenant-Network-Message-Id: 
 cd4f7468-6e0e-4681-202e-08d905ad3d0a
X-MS-Exchange-CrossTenant-AuthSource: SN6PR12MB2718.namprd12.prod.outlook.com
X-MS-Exchange-CrossTenant-AuthAs: Internal
X-MS-Exchange-CrossTenant-OriginalArrivalTime: 22 Apr 2021 16:39:46.2380
 (UTC)
X-MS-Exchange-CrossTenant-FromEntityHeader: Hosted
X-MS-Exchange-CrossTenant-Id: 3dd8961f-e488-4e60-8e11-a82d994e183d
X-MS-Exchange-CrossTenant-MailboxType: HOSTED
X-MS-Exchange-CrossTenant-UserPrincipalName: 
 WHpbJYThSqjFrrT/ZcYKSEFvX1ozm4sPthYUYc/LO4TprekaYHCCLZFET2kOYOPmQfbs9Ai6+Su9j+BXo/2FSA==
X-MS-Exchange-Transport-CrossTenantHeadersStamped: SA0PR12MB4495
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Fixes: 15fb7de1a7f5 ("KVM: SVM: Add KVM_SEV_RECEIVE_UPDATE_DATA command)"
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Brijesh Singh <brijesh.singh@amd.com>
---
 Documentation/virt/kvm/amd-memory-encryption.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/virt/kvm/amd-memory-encryption.rst b/Documentation/virt/kvm/amd-memory-encryption.rst
index 07f8633b61cd..bc8e5b7f0c35 100644
--- a/Documentation/virt/kvm/amd-memory-encryption.rst
+++ b/Documentation/virt/kvm/amd-memory-encryption.rst
@@ -395,7 +395,7 @@ On success, the 'handle' field contains a new handle and on error, a negative va
 For more details, see SEV spec Section 6.12.
 
 16. KVM_SEV_RECEIVE_UPDATE_DATA
-----------------------------
+-------------------------------
 
 The KVM_SEV_RECEIVE_UPDATE_DATA command can be used by the hypervisor to copy
 the incoming buffers into the guest memory region with encryption context

From patchwork Thu Apr 22 16:38:35 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Brijesh Singh <brijesh.singh@amd.com>
X-Patchwork-Id: 12218915
Return-Path: <kvm-owner@kernel.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-18.8 required=3.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,HEADER_FROM_DIFFERENT_DOMAINS,INCLUDES_CR_TRAILER,
	INCLUDES_PATCH,MAILING_LIST_MULTI,MSGID_FROM_MTA_HEADER,SPF_HELO_NONE,
	SPF_PASS,USER_AGENT_GIT autolearn=ham autolearn_force=no version=3.4.0
Received: from mail.kernel.org (mail.kernel.org [198.145.29.99])
	by smtp.lore.kernel.org (Postfix) with ESMTP id 4A1C1C433B4
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 16:39:52 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 20D3861421
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 16:39:52 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S238067AbhDVQkZ (ORCPT <rfc822;kvm@archiver.kernel.org>);
        Thu, 22 Apr 2021 12:40:25 -0400
Received: from mail-mw2nam10on2053.outbound.protection.outlook.com
 ([40.107.94.53]:20929
        "EHLO NAM10-MW2-obe.outbound.protection.outlook.com"
        rhost-flags-OK-OK-OK-FAIL) by vger.kernel.org with ESMTP
        id S236780AbhDVQkX (ORCPT <rfc822;kvm@vger.kernel.org>);
        Thu, 22 Apr 2021 12:40:23 -0400
ARC-Seal: i=1; a=rsa-sha256; s=arcselector9901; d=microsoft.com; cv=none;
 b=aRdzkie25Whp+J4wfivUK+DeybKCu+dE1W484nMPVftJ4NAysTv3d9R3NZ4Q+YA4Idu+jF7iREu9xvNv9giEpJXe5O3aESH9+H2rLCpkq/Lzk/iiCvk6b7CHrDdCbTrKxyBn2jLCk48dHXLFma/jUH0Zf3ilBMm+nswzidIsPaVG7eVwbN8oPvzmVL2iUG/pds2M7p61ETs3RiqjS8NQ4zfMBKAZKzV8a73dHr28fN5mnADRP7Wgq4l2OZVeYz8oKJI6WGi+6IaeLjTvjxr2ANpZsMbpaURPTakq4roeTzLtsOFtGsijzX+b/4mLfeAE2vM7fhlFsMNRaIYM4AmHhA==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=microsoft.com;
 s=arcselector9901;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-SenderADCheck;
 bh=7VACaQmzjscsfqBWYkWKr4G4uGiFzlyPe0+NoL3EeEU=;
 b=dB4rU23Id+yOz9bQm5Vye2Sh9AirZg8UxcnmFwExNuqQA2DA0qxiqr+AgKiYHOO25gcfNfha7c3CcE51EKh31sp2xx8iLVcqVsF/U2cKhyML5lRHbNOT2Sq0F8ap1LqJxp+M26IWTl2j1py2qjNqKPhqwgPqKZ8MnpDp1MaL5UGp59+jhFqIgQEY9m/7k6fFl8dhghctoDwSw6VfXAWM4dT3sWniahelqtUluuxGT66ADHazN62bsta/g69CLw2LTxNSjo82vjk4mGHzIpjVx7B5DsvczjCpxyCU8M+TjCay5YAt6LIVGBsPU5SxRNv9hO4WMnhLKM6Eilsqhr3Slg==
ARC-Authentication-Results: i=1; mx.microsoft.com 1; spf=pass
 smtp.mailfrom=amd.com; dmarc=pass action=none header.from=amd.com; dkim=pass
 header.d=amd.com; arc=none
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=amd.com; s=selector1;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-SenderADCheck;
 bh=7VACaQmzjscsfqBWYkWKr4G4uGiFzlyPe0+NoL3EeEU=;
 b=sik7MsZvZ3ZMnnLEc5IHSS6I/vRyVjzF1Ff3suMs/jAocUCGPA02YbmNGomSAsFRcxcWdtntR5VkmhqIxZW87J+LpkPzURxwpIsGiAMZEUK8ixSKL2S/QGuvxvUbBTA8V8K/o9DKRcCBMOIT/itF+IzRQxbDSaFb3T6qOB2jlls=
Authentication-Results: redhat.com; dkim=none (message not signed)
 header.d=none;redhat.com; dmarc=none action=none header.from=amd.com;
Received: from SN6PR12MB2718.namprd12.prod.outlook.com (2603:10b6:805:6f::22)
 by SN1PR12MB2446.namprd12.prod.outlook.com (2603:10b6:802:26::17) with
 Microsoft SMTP Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.4065.21; Thu, 22 Apr
 2021 16:39:47 +0000
Received: from SN6PR12MB2718.namprd12.prod.outlook.com
 ([fe80::9898:5b48:a062:db94]) by SN6PR12MB2718.namprd12.prod.outlook.com
 ([fe80::9898:5b48:a062:db94%6]) with mapi id 15.20.4065.023; Thu, 22 Apr 2021
 16:39:47 +0000
From: Brijesh Singh <brijesh.singh@amd.com>
To: pbonzini@redhat.com, kvm@vger.kernel.org
Cc: linux-kernel@vger.kernel.org, sfr@canb.auug.org.au,
        Ashish.Kalra@amd.com, Brijesh Singh <brijesh.singh@amd.com>
Subject: [PATCH 3/4] docs: kvm: fix underline too short warning for
 KVM_SEV_RECEIVE_FINISH
Date: Thu, 22 Apr 2021 11:38:35 -0500
Message-Id: <20210422163836.27117-4-brijesh.singh@amd.com>
X-Mailer: git-send-email 2.17.1
In-Reply-To: <20210422163836.27117-1-brijesh.singh@amd.com>
References: <20210422163836.27117-1-brijesh.singh@amd.com>
X-Originating-IP: [165.204.77.1]
X-ClientProxiedBy: SN2PR01CA0059.prod.exchangelabs.com (2603:10b6:800::27) To
 SN6PR12MB2718.namprd12.prod.outlook.com (2603:10b6:805:6f::22)
MIME-Version: 1.0
X-MS-Exchange-MessageSentRepresentingType: 1
Received: from sbrijesh-desktop.amd.com (165.204.77.1) by
 SN2PR01CA0059.prod.exchangelabs.com (2603:10b6:800::27) with Microsoft SMTP
 Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.4065.20 via Frontend
 Transport; Thu, 22 Apr 2021 16:39:46 +0000
X-MS-PublicTrafficType: Email
X-MS-Office365-Filtering-Correlation-Id: d4f29026-e7ab-4d99-0554-08d905ad3d53
X-MS-TrafficTypeDiagnostic: SN1PR12MB2446:
X-MS-Exchange-Transport-Forked: True
X-Microsoft-Antispam-PRVS: 
 <SN1PR12MB2446E33C2C8EFEB5DA8B00DBE5469@SN1PR12MB2446.namprd12.prod.outlook.com>
X-MS-Oob-TLC-OOBClassifiers: OLM:5516;
X-MS-Exchange-SenderADCheck: 1
X-Microsoft-Antispam: BCL:0;
X-Microsoft-Antispam-Message-Info: 
 hPTAe5YuthKvY+NM4fvZfHs/eSqU8bedNqv7fzvtsPky4jnWKoY0HhRp8EzVf5ZDE6lvw39zcklzLS0rFoveL9qA+T4iE5n02pWaXmD5rOVF2lLdjG6E+SPi9IkJTvUx4VyDXth1yMqm5DEw8m1KLJTK49k+Ds0gdPyH7MpagdGy5UUKq3kSZSvWpM+rcFGD+lLyZO9UZlic6HGVXW0+wnwLFrqDz/AHuUyHp9V+i0mkJto78VXMB+fIoQ0UnxbTIfS6kJO9HYLnrzst8wQ3X6RNLaFHJJy8D+igsONPHcvd514GafwEeHKyRplKPmNw2Ye0GxwWE47X3jQcsrqiMbQujpootNRNDFiRAPRofQebLuji6AJjW4S86aTA9x/zu12V75ETgAURdpjqJ2INEEWSzNx9YQVvWVwpwHlXkbEPyHPV/oGK+SWKEhC4IhFGrdIGZhi2FZlVNitwRSLoYaG4FEd7POIDQEJgE3urtkE/9wKFYoYP2UPM+I7laPEqpy1Uh5HbhcviHh8P/5Aa/bHTQUleagycEK9M399kU4tPp2kbpm5luir6zdJbWas9cUIZ6Y1HMfwa1yi3bpet0tLQfInfYLh6qkjiHCwzliyfbG/h2nib25Ip6dhYa6/RsgjYp40seMeZowGXlAB7Mg==
X-Forefront-Antispam-Report: 
 CIP:255.255.255.255;CTRY:;LANG:en;SCL:1;SRV:;IPV:NLI;SFV:NSPM;H:SN6PR12MB2718.namprd12.prod.outlook.com;PTR:;CAT:NONE;SFS:(4636009)(376002)(346002)(396003)(136003)(39850400004)(366004)(66476007)(86362001)(2906002)(186003)(4744005)(1076003)(66556008)(5660300002)(16526019)(6666004)(7696005)(6486002)(66946007)(38100700002)(478600001)(8936002)(36756003)(38350700002)(26005)(8676002)(52116002)(316002)(2616005)(956004)(44832011)(4326008)(83380400001);DIR:OUT;SFP:1101;
X-MS-Exchange-AntiSpam-MessageData: 
 +ul/8fPsGyB2L3PVjW5KXYSRXvgJMOzDRdvX+9ZeUHjVY0GxAxsUaZOR/04LlDWKh3IzGBXtuvYuFWQRBBa/Tw2kw2UuK5HMMaX7MBY6iEcZlHKVGnuo/kfSdadASI0a2TuVWYwQLXt21bZHvzpJIXowprXsBUAKXqzpIeSv31J3jIDpwtSkBe0M2PjHD0p9fCvFqdgpYiQgYmd8SynlgkAcYY1kGDkXS0HyNaUG6jOwzOIElSjwNOkpNxT+MJE8gV3q7VS9prM1fenZl1QKtLJ+Pw5hAdLcN6K8ZiPJFjbe1T5FKe4W2g+akBwYcwZCHU8SVCpnPiyoLB7UUTA6GuhT775p4M7ptBmuep3u6wuTIKwGz/O+AQsCe9Zbu7bjMOyb//Bc0DWi2TKFOOEJsWDX83Ompdqmw+Er6j15+bblTae+JtFbyvIypATTI8qt2zlnLOcfYxAouhplb1+OaubanD910WnrULhRIgxGCNeDep5LRNnW5wf/9/6tNHLgY39ALrWzV6xrmqBxW08vGJZCU3sSRRMg093g1J/BvpbEwB42qCQG+PPOK8dRVh8pPQ/QYMajpH8QYXbmG1NykMhnyMW7Z4HZAYJL85Mwv1Ngm84Kjzak1IpuyK9rjwP9ehSau3cEy+KkwRHst4W0pXRooVVI5IJMgTTFoq4VRFivkymh1tPd543hPp34eL6BIhPepsdfTx3pgkfZ8GUZPTIirBAoo/mV/yeOkvp4x4lCazyQEvdOAfrGmmQ/Ag4iU23eQSr2V9IJX+mPJb/nvjXLU6pIhWhU7ieQYAee81yJcrS2Jd+lZBfvO2ai5ix4lXuBz9uxJg30s02Bu8qcxLo6lM68j+59TDTlkn8+FH2FonH96hEEuzo+2nGLl/hzzd1/64Hk5ac5sualR1fGzYlrBswXXmwEjVPVfPrbmnr8m6Y/hnM8PFWTJR8EAgz1WuZmg7cly9HRyDa0et7p/gKunXx1nkwrlhlkGiqfiDPdPe1HboZjbIfrzcrEI4MZFErLXuJKJTjZlRugmgjVDktsvfo/cLyhbcqdDEbDYCoJ9xMiV0o173gRYiLs+pkHtQMLtz3WYIE7r5bAc8x6bMdPvLkN+qd/ZoQdw9hz1LFjt2wTKlgEcFcMptDQcziQVxQVt2/V65eaT+vR/j8R4Kkb6fHigwCvHiVyv/7IRtRfTuo1wfNu3kow1n2BJ2tPCGWKHRWzwblNHBeQun3rDSDo7l+Rfq/CDix0QHpyuDgQuplVnARBJvnkYOjsy+RVR+j+s26iEiwzZtI5vlKpoT9BHw92zQ0/CvjoO5UmKLEyZaKETukQosOJdT3pOBXo
X-OriginatorOrg: amd.com
X-MS-Exchange-CrossTenant-Network-Message-Id: 
 d4f29026-e7ab-4d99-0554-08d905ad3d53
X-MS-Exchange-CrossTenant-AuthSource: SN6PR12MB2718.namprd12.prod.outlook.com
X-MS-Exchange-CrossTenant-AuthAs: Internal
X-MS-Exchange-CrossTenant-OriginalArrivalTime: 22 Apr 2021 16:39:46.7057
 (UTC)
X-MS-Exchange-CrossTenant-FromEntityHeader: Hosted
X-MS-Exchange-CrossTenant-Id: 3dd8961f-e488-4e60-8e11-a82d994e183d
X-MS-Exchange-CrossTenant-MailboxType: HOSTED
X-MS-Exchange-CrossTenant-UserPrincipalName: 
 r0uylHc5MsbYtqW7QtnNpwYOgxbfOur+ib4/0qSQkczASb6uVzsL9z10ThhPRsfyGyTOvW1tmt4uicgHKCam0Q==
X-MS-Exchange-Transport-CrossTenantHeadersStamped: SN1PR12MB2446
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Fixes: 6a443def87d2 ("KVM: SVM: Add KVM_SEV_RECEIVE_FINISH command")
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Brijesh Singh <brijesh.singh@amd.com>
---
 Documentation/virt/kvm/amd-memory-encryption.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/virt/kvm/amd-memory-encryption.rst b/Documentation/virt/kvm/amd-memory-encryption.rst
index bc8e5b7f0c35..eef3eff799fe 100644
--- a/Documentation/virt/kvm/amd-memory-encryption.rst
+++ b/Documentation/virt/kvm/amd-memory-encryption.rst
@@ -419,7 +419,7 @@ Returns: 0 on success, -negative on error
         };
 
 17. KVM_SEV_RECEIVE_FINISH
-------------------------
+--------------------------
 
 After completion of the migration flow, the KVM_SEV_RECEIVE_FINISH command can be
 issued by the hypervisor to make the guest ready for execution.

From patchwork Thu Apr 22 16:38:36 2021
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Brijesh Singh <brijesh.singh@amd.com>
X-Patchwork-Id: 12218919
Return-Path: <kvm-owner@kernel.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-18.8 required=3.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,HEADER_FROM_DIFFERENT_DOMAINS,INCLUDES_CR_TRAILER,
	INCLUDES_PATCH,MAILING_LIST_MULTI,MSGID_FROM_MTA_HEADER,SPF_HELO_NONE,
	SPF_PASS,USER_AGENT_GIT autolearn=ham autolearn_force=no version=3.4.0
Received: from mail.kernel.org (mail.kernel.org [198.145.29.99])
	by smtp.lore.kernel.org (Postfix) with ESMTP id 16484C433B4
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 16:39:55 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id DB73C6141C
	for <kvm@archiver.kernel.org>; Thu, 22 Apr 2021 16:39:54 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S238140AbhDVQk2 (ORCPT <rfc822;kvm@archiver.kernel.org>);
        Thu, 22 Apr 2021 12:40:28 -0400
Received: from mail-mw2nam10on2053.outbound.protection.outlook.com
 ([40.107.94.53]:20929
        "EHLO NAM10-MW2-obe.outbound.protection.outlook.com"
        rhost-flags-OK-OK-OK-FAIL) by vger.kernel.org with ESMTP
        id S237972AbhDVQkY (ORCPT <rfc822;kvm@vger.kernel.org>);
        Thu, 22 Apr 2021 12:40:24 -0400
ARC-Seal: i=1; a=rsa-sha256; s=arcselector9901; d=microsoft.com; cv=none;
 b=DVK8sJ6ze9eXkRFPQLI3PKepXKswusTDEDvlj9H4EUa2U7NJ6ANU9aVG6MqxroJcV2wlZh9QqEgX/yY4w0XdR5NWKccl6o5eD+wU3eRxr/O6bmGJTVjKc1m/MUIbYrPT7uSZ+VXHz5qXIkhBhf7UoQYzoHqfQN7okAXaAV8lQq3W83lNNqRNUpan1sS6Tu1XqPpRt0sYn6+kNnrvvu/zKe1LGoxZAuh+xADk+jioXZbwGNJrz+6BRShQpI1gy98NX1Eyc9NhUgdXzSsCICybEbrBF0Q3yRU+pjz5ljIZ7ClRPsKV/9SKetsRuZV4S2yVMAC16hfwDgawiNNf2USy7g==
ARC-Message-Signature: i=1; a=rsa-sha256; c=relaxed/relaxed; d=microsoft.com;
 s=arcselector9901;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-SenderADCheck;
 bh=srVjAMBmxP5PLZiGofLL8o3oZArSe1UjnDwIqbxVB0w=;
 b=XixCSndnr0WvJ0DOE8U8i3zkW8pKfwpZJ/krKgYw8EbuyQ2c5NAdO9KA3HEiwksUC/bqD1Wtu9wnHWvIiFF+/U5WNibw4X54vqPGqAxo+8S87J+Obmi9Bv7OIAWl+/5dJii5ewO/oDej7FCGzrb8SEjDnn/4tQkQnU5cka24TSofbvgZRVYQl3JAznUDCa1eYnVDh3Cqz8nF+yNlM8+IL1WnyTdYVWF7aATxvZmhJMOQz9X0PXH5uicH05W+t8m7zNAdviEZpkK2ThU1UamWbz9C8BYXsynp5XESXRt+Fm96srdfmgsH9nhX8fD7DxiNvEiLM0GWpJH4MW5o1ubmFA==
ARC-Authentication-Results: i=1; mx.microsoft.com 1; spf=pass
 smtp.mailfrom=amd.com; dmarc=pass action=none header.from=amd.com; dkim=pass
 header.d=amd.com; arc=none
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=amd.com; s=selector1;
 h=From:Date:Subject:Message-ID:Content-Type:MIME-Version:X-MS-Exchange-SenderADCheck;
 bh=srVjAMBmxP5PLZiGofLL8o3oZArSe1UjnDwIqbxVB0w=;
 b=oII7xo8tPkbuXgU22Rt+6qMufpGYCCo4uw3jVsm2ztUJ4C4+I7aYf2RrTtssQs9Hz6JXuJ8KnG5dDr6uzNdtK0PLD1R+dpuves9kH8BWV1hGj8vw4fZqxT3sAylHTnLOohFjSdaUYlmy/WZewflSA8dPxSLSV5RIgiFTzUcg8Xc=
Authentication-Results: redhat.com; dkim=none (message not signed)
 header.d=none;redhat.com; dmarc=none action=none header.from=amd.com;
Received: from SN6PR12MB2718.namprd12.prod.outlook.com (2603:10b6:805:6f::22)
 by SN1PR12MB2446.namprd12.prod.outlook.com (2603:10b6:802:26::17) with
 Microsoft SMTP Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.4065.21; Thu, 22 Apr
 2021 16:39:47 +0000
Received: from SN6PR12MB2718.namprd12.prod.outlook.com
 ([fe80::9898:5b48:a062:db94]) by SN6PR12MB2718.namprd12.prod.outlook.com
 ([fe80::9898:5b48:a062:db94%6]) with mapi id 15.20.4065.023; Thu, 22 Apr 2021
 16:39:47 +0000
From: Brijesh Singh <brijesh.singh@amd.com>
To: pbonzini@redhat.com, kvm@vger.kernel.org
Cc: linux-kernel@vger.kernel.org, sfr@canb.auug.org.au,
        Ashish.Kalra@amd.com, Brijesh Singh <brijesh.singh@amd.com>
Subject: [PATCH 4/4] docs: kvm: fix inline emphasis string warning for
 KVM_SEV_SEND_START
Date: Thu, 22 Apr 2021 11:38:36 -0500
Message-Id: <20210422163836.27117-5-brijesh.singh@amd.com>
X-Mailer: git-send-email 2.17.1
In-Reply-To: <20210422163836.27117-1-brijesh.singh@amd.com>
References: <20210422163836.27117-1-brijesh.singh@amd.com>
X-Originating-IP: [165.204.77.1]
X-ClientProxiedBy: SN2PR01CA0059.prod.exchangelabs.com (2603:10b6:800::27) To
 SN6PR12MB2718.namprd12.prod.outlook.com (2603:10b6:805:6f::22)
MIME-Version: 1.0
X-MS-Exchange-MessageSentRepresentingType: 1
Received: from sbrijesh-desktop.amd.com (165.204.77.1) by
 SN2PR01CA0059.prod.exchangelabs.com (2603:10b6:800::27) with Microsoft SMTP
 Server (version=TLS1_2,
 cipher=TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384) id 15.20.4065.20 via Frontend
 Transport; Thu, 22 Apr 2021 16:39:46 +0000
X-MS-PublicTrafficType: Email
X-MS-Office365-Filtering-Correlation-Id: fd90b252-ee90-46d4-4e4e-08d905ad3d9b
X-MS-TrafficTypeDiagnostic: SN1PR12MB2446:
X-MS-Exchange-Transport-Forked: True
X-Microsoft-Antispam-PRVS: 
 <SN1PR12MB24464A43BFA4DF96EA55BF71E5469@SN1PR12MB2446.namprd12.prod.outlook.com>
X-MS-Oob-TLC-OOBClassifiers: OLM:2733;
X-MS-Exchange-SenderADCheck: 1
X-Microsoft-Antispam: BCL:0;
X-Microsoft-Antispam-Message-Info: 
 Ohq5+eLemu3Pc9v9HJsS2fuGF70SI7rgA8b53F9ekOxtR+hmQJb1c/99JCYxXz0UyHkRzC+AgKZn8fL/Yq6srs5YNG235fZtG/HRI08pdREgBtMSwYA7FCJZbCrL/88jYImixKUNS0rZ/+bZjm7xQjyJwHNIE4OmYDCYKmVwCg0qoMgCIunV90jpHfzeR2Vyn4U8CCpgG2S2+EjX5ZQc5bBIy+Rs7tpKAPjywvoAdT1ldrGtQ1X6362on6aKejaiNOtHdscDoGbDcd3K7WGpqj9l1EnwI6hkAT4dORvu1iw1E1eo7QnOBVhCf4jCASo0NGKvunQFaxRztGzUuCJuu5UABKFDQtniDLfxIhxIC/NM6h0VBaG6WV4Sx2ADXrWrPfKAEytWH+xchaoGVwdvilowhN8ACYpM2fk3jKt+CHbQjftoTfmENs7Aoy8tw43APj7AFyBe60Zem+9SKjdcvUYiwQFOHR8yLvG+HH1+HiW2M8cfGhW9os6vn3jn9FCpwU62pyTFer5o0PWIDo49JarCiJwkhwpaRju3S+RxOXwHOVWPn/IV3nZqiOoip59U1pJyAPgjF6QztiDzmrJzorWMJMic0blPuepN/MFI9VqABa4otyJi+gnLQjit9IhjHMC/la1/c63SC54iWL/Kxqf3HMptlzRYQ02jE1EaRcs=
X-Forefront-Antispam-Report: 
 CIP:255.255.255.255;CTRY:;LANG:en;SCL:1;SRV:;IPV:NLI;SFV:NSPM;H:SN6PR12MB2718.namprd12.prod.outlook.com;PTR:;CAT:NONE;SFS:(4636009)(376002)(346002)(396003)(136003)(39850400004)(366004)(66476007)(86362001)(2906002)(186003)(4744005)(1076003)(66556008)(5660300002)(16526019)(6666004)(7696005)(6486002)(66946007)(38100700002)(478600001)(8936002)(36756003)(38350700002)(26005)(8676002)(52116002)(316002)(2616005)(956004)(44832011)(4326008)(83380400001);DIR:OUT;SFP:1101;
X-MS-Exchange-AntiSpam-MessageData: 
 4ICE8jCKXg/tstnVgBJzSnLunCe6F+Fy8eD9Q3bldSZU7ADcbgixnd5tG7mTjPJzGcKYCLQzvLBgx4lnue/0JIxg4xCUzmFRl03j9wtyEd5TyV+4XyzOGmG26JGZgW0KE53vLh+QU9i1+T4odN5jR4V2SR1Mnhq6Y58Op+InzlJRbTJ4MtVFOM4JEjyepgq7MYqHaboKtrh8kprX2MaZnBIoxQ37k+kjKsK+UYsTReXPFXU6iDbSOjvRvrs/KQGFx1WFFZJmR1ullpfWT65cXz+WQz0bRBLhQWUkSAaJ0BlQWx8ZICyNC8kA4PCj15XZyWekErP/GIW/p5DZ2qF1iGsqk3HQIvn560bxy/2hjXDlh/zkjgY36NKUPy3NDrYcA8CqpvwvksHARCPNcv64HJG7AFPAQza+oQ1Nuu9OMvDgx/cFLmjm1roj1H+APW5+yj4oYWzB4B+vaoS+2Y8srtBYoGCITqcv8ZCVYEbQAGMu2pY5aZLzqMBdjIM2M9B7xyfnQTa0cYpEQhP+HnELcA9Hq3IFVyCvaZ3MHFCMnJDUl2bqZurZUN3q5PvvopaFCV1nT3LXYz3phRwBgVOSoQsDobu54w8IsEkMU4tIUQ/E9lQsNoRpAfwy7/TCbprRE8MfCJt2ubj66TWPFAl+d90UCbXJtI2Uhj8SaperNqTGO51kCev3yeM1eTld0EM8OyG1DDyf8nu2aNfwMKHxSxozhgE/QyVBDAjuWsJ22/l2t1RrbeIkfehu6T4vM7KMEaLQ+YMsYLd4CaxB3J41ymHS0k4M3KolFyQ8p3swFErJFKbzQJyPcxzHonu6L1xPSmGDuJXWWHero1W74If5eecEtZxHgsJMhaNXvvZn+NjiUvjOxPZpfRM/8P33ClEIpNx27iDYdwSuyNEWhriTxy36TyatmNw4KWjasT+cke4omk87JmUfZ0+yJqL4EQKpc0R/LIQTPHhPXR30/EEN6ahrazJ+J72w3rPsxMhVWZnntriz1E/qcl29A0BU2UHPCpHdHVjmoUjw9qf1fWUeq0kNePGsRI7LJlLoPUAo83rk6ey/5BAYqLMJLs1aTN/AUFz+fQmwX3Ae03QVoYGCBUGNelrsNK6wZQVdaUvKWemgkvxu8yhJgtvTEPYziY2Fe0E+fh/8zCtFLL9MFIn/4PyF6EPqLYDnpsE3LzSZVqOzXaOMEYnxJAGc/bqUMc21zy8xpxOw9v6vqK4EU6sS9soK5KvN8GwE2pu2EbagH2jfKFCjFwhB6GWXV3fFe/kybWPJVnrClBYGCoV6YE1KD2j5BsRZ4pXgcQU44lykuHuxYiRJ3MS7+7OfsrTkqfT2
X-OriginatorOrg: amd.com
X-MS-Exchange-CrossTenant-Network-Message-Id: 
 fd90b252-ee90-46d4-4e4e-08d905ad3d9b
X-MS-Exchange-CrossTenant-AuthSource: SN6PR12MB2718.namprd12.prod.outlook.com
X-MS-Exchange-CrossTenant-AuthAs: Internal
X-MS-Exchange-CrossTenant-OriginalArrivalTime: 22 Apr 2021 16:39:47.1574
 (UTC)
X-MS-Exchange-CrossTenant-FromEntityHeader: Hosted
X-MS-Exchange-CrossTenant-Id: 3dd8961f-e488-4e60-8e11-a82d994e183d
X-MS-Exchange-CrossTenant-MailboxType: HOSTED
X-MS-Exchange-CrossTenant-UserPrincipalName: 
 HxPSZhAOjaEFnVjcB/XnSzJTeFK1TrpajuPMKH0OpdKubGJ/eUD+vMGs+NNNBH9+hFeipnJBsbG+uJWzk2f8QQ==
X-MS-Exchange-Transport-CrossTenantHeadersStamped: SN1PR12MB2446
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Fix the following warning seen with 'make htmldocs'

WARNING: Inline emphasis start-string without end-string

Fixes: 4cfdd47d6d95 ("KVM: SVM: Add KVM_SEV SEND_START command")
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Brijesh Singh <brijesh.singh@amd.com>
---
 Documentation/virt/kvm/amd-memory-encryption.rst | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/virt/kvm/amd-memory-encryption.rst b/Documentation/virt/kvm/amd-memory-encryption.rst
index eef3eff799fe..b45f2358e143 100644
--- a/Documentation/virt/kvm/amd-memory-encryption.rst
+++ b/Documentation/virt/kvm/amd-memory-encryption.rst
@@ -304,6 +304,7 @@ Parameters (in): struct kvm_sev_send_start
 Returns: 0 on success, -negative on error
 
 ::
+
         struct kvm_sev_send_start {
                 __u32 policy;                 /* guest policy */
 
